evalml/tests/data_checks_tests/test_data_checks.py

```diff
@@ -160,7 +160,7 @@ def validate(self, X, y):
         },
     ).to_dict(),
     DataCheckWarning(
-        message="Column(s) 'lots_of_null', 'nullable_integer', 'nullable_bool', 'natural_language_nan', 'nan_dt_col' have null values",
```

Comment:
Checking only a subset of columns means no more detection of datetime cols with nan values, see comment in `evalml/data_checks/null_data_check.py`